build.sbt

```diff
@@ -23,11 +23,10 @@ val avroVersion = "1.9.2"
 val bigqueryVersion = "v2-rev20191211-1.30.3"
 val catsVersion = "2.0.0"
 val datastoreVersion = "1.6.3"
-val grpcApiVersion = "1.17.0"
```

Comment:
Why was this needed? Everything works fine for me without it.

Reply:
I was seeing issues where a class in the package `com.google.type` was causing a NoClassDefFoundError in the datastore package - would fail when I tried to run all tests locally. Adding this jar which contains that class fixed it for me. see https://github.com/spotify/magnolify/pull/51/commits/e098934305400a5c4e950864137ffedc6736aa50

Reply:
Yeah weird. I saw the dependency, but never had any problem with datastore without it.